unitTests/ui/CategoryFacet/CategoryFacetTest.ts

```diff
@@ -276,6 +276,15 @@ export function CategoryFacetTest() {
         }
       }
 
+      function splitSelectableParents() {
+        const parentValuesLabel = $$(test.cmp.element).findAll('.coveo-category-facet-parent-value label');
+        const splitOnSelectableOrNot = partition(parentValuesLabel, parentLabel => $$(parentLabel).hasClass('coveo-selectable'));
```

Comment:
Could use destructuring here: `const { selectables, notSelectable } = partition(...)`
Should the second name be plural for consistancy? maybe `notSelectables` or `unSelectables`?

Reply:
Since it's an array wouldn't it be `const [ selectables, notSelectable ] = partition(...)`